README.md

```diff
@@ -50,10 +52,47 @@ In addition to `live_view_native` you may want to include some additional librar
 {:live_view_native_live_form, "~> 0.3.0-rc.3"}
 ```
 
+The versions above may be subject to change, so make sure to check the library documentation for the latest version.
+* [LiveView Native Versions](https://hex.pm/packages/live_view_native/versions)
+* [LiveView Native Stylesheet Versions](https://hex.pm/packages/live_view_native_stylesheet/versions)
+* [LiveView Native SwiftUI Client Versions](https://hex.pm/packages/live_view_native_swiftui/versions)
+* [LiveView Native Live Form Versions](https://hex.pm/packages/live_view_native_live_form/versions)
+
 Then run:
 
 ```
 $ mix lvn.setup
 ```
 
 and follow the instructions on how to complete the setup process.
+
+## Main Branch
+
+To try out new features or get the latest bugfixes you can also use the `main` branch for LiveView Native dependencies.
+
+```elixir
+{:live_view_native, github: "liveview-native/live_view_native", override: true},
+{:live_view_native_stylesheet, github: "liveview-native/live_view_native_stylesheet"},
+{:live_view_native_swiftui, github: "liveview-native/liveview-client-swiftui"},
+{:live_view_native_live_form, github: "liveview-native/liveview-native-live-form"}
+```
+
+However, be aware that active changes may make your development experience unstable.
+
+## Native Clients
+
+LiveView Native enables native client frameworks such as the [LiveView Native SwiftUI Client](https://github.com/liveview-native/liveview-client-swiftui) and the [LiveView Native Jetpack Client (In Progress)](https://github.com/liveview-native/liveview-client-jetpack).
+
+### SwiftUI Client
+
+To create your first LiveView Native LiveView, go to the [LiveView Native SwiftUI Client Usage](https://github.com/liveview-native/liveview-client-swiftui?tab=readme-ov-file#usage) guide on GitHub. Alternatively, you can read the [LiveView Native SwiftUI Documentation] where you can find documentation, examples, and interactive Livebook guides. 
```

Comment:
I don't see why we need to include client-specific sections in here, that's for those docs

Reply:
My thinking was it's not immediately obvious to someone new to LVN that these client libraries exist, or where to find resources for them.

Reply:
yeah I think the client matrix noted above should cover that. We can just link out to them

Reply:
I've removed them, but kept a list referencing them to improve discoverability.

Reply:
no worries :)